app/src/main/java/com/github/fribourgsdp/radio/DisplayLyricsActivity.kt

```diff
@@ -0,0 +1,36 @@
+package com.github.fribourgsdp.radio
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import okhttp3.OkHttpClient
+
+
+class DisplayLyricsActivity : AppCompatActivity() {
+    private var songTextView : TextView? = null
+    private var artistTextView : TextView? = null // TODO: right way to declare textviews ?
+    private var resultsTextView : TextView? = null
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_display_lyrics)
+        songTextView = findViewById(R.id.songEditView)
+        artistTextView = findViewById(R.id.artistEditView)
+        resultsTextView = findViewById(R.id.resultsTextView)
+        val goButton : Button = findViewById(R.id.button1)
+        goButton.setOnClickListener {
+            run {
+                val songName: String = songTextView?.text.toString()
+                val artistName: String = artistTextView?.text.toString()
+                // TODO: handle exception
```

Comment:
Add exception handler :)

Reply:
Actually it was an old TODO comment I forgot to erase, exception handling was added in commit b7fd8cf.